api-test/src/main/java/io/mosip/testrig/apirig/utils/MimotoUtil.java

```diff
@@ -91,4 +91,12 @@ public static String getOTPFromSMTP(String inputJson, TestCaseDTO testCaseDTO) {
 		return inputJson;
 	}
 	
+	public static String getClientIdSection(String baseURL) {
```

Comment:
have we not handled the sunbird usecase ?

Reply:
Yes, Vijay. I changed it to accommodate sunbird also